group_vars/mons.yml.sample

```diff
@@ -43,14 +43,16 @@ dummy:
 
 #crush_rule_hdd:
 #  name: HDD
-#  root: HDD
+#  root: default
 #  type: host
+#  class: hdd
 #  default: false
 
 #crush_rule_ssd:
 #  name: SSD
-#  root: SSD
+#  root: deufalt
```

Comment:
typo:
```suggestion
#  root: default
```
in anycase you mustn't edit sample files directly, please read https://github.com/ceph/ceph-ansible/blob/master/CONTRIBUTING.md#sample-files